### PR TITLE
chore: update description of golang resource by programiz

### PR DESCRIPTION
**Pull request by subhadeepbanerjee2003**
## Fixes Issue

- I have changed the description for `"Learn Go Programming" tutorial by programiz`

**Closes issue id #1613**

## Changes proposed

 ```
 },
  {
    "name": "Learn Go Programming",
    "description": "This playlist contains all the concepts of Golang along with two mini projects",
    "description": "Our Go tutorials will guide you to learn Go Programming one step at a time",
    "url": "https://www.programiz.com/golang",
    "category": "languages",
    "subcategory": "golang"
```

## Screenshots
Screenshot from 2023-08-23 21-38-06

## Note to reviewers

- **`NOTE` :: If you want to modify the description please let me know.**

### CODE REVIEW

**Comment by subhadeepbanerjee2003** [comment]
@Anmol-Baranwal @CBID2 check my pr and the code base and let me know if there is any scope of improvement?

**Comment by Anmol-Baranwal** [comment]
I will check the resource later. Then I will let you know.

> What was wrong in the previous description?

**Comment by subhadeepbanerjee2003** [comment]
@Anmol-Baranwal the previous description was the description of the other tutorial which was unfortunately added.

**Comment by Anmol-Baranwal** [comment]
@CBID2 @rupali-codes 

Please review this.

**Review by k-deepak04 — APPROVED** [review]

File: database/languages/golang.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
   },
   {
     "name": "Learn Go Programming",
-    "description": "This playlist contains all the concepts of Golang along with two mini projects",
+    "description": "Our Go tutorials will guide you to learn Go Programming one step at a time",
```
Comment (CBID2) [inline database/languages/golang.json:25]:
```suggestion
    "description": "Our Go tutorials will guide you in learning the programming one step at a time",
```

**Comment by subhadeepbanerjee2003** [comment]
@CBID2 yah I have reviewed it and done the necessary.

File: database/languages/golang.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
   },
   {
     "name": "Learn Go Programming",
-    "description": "This playlist contains all the concepts of Golang along with two mini projects",
+    "description": "Our Go tutorials will guide you in learning the programming one step at a time",
```
Comment (CBID2) [inline database/languages/golang.json:25]:
```suggestion
    "description": "Our Go tutorials will guide you in learning the programming language one step at a time",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Still needs improvement @subhadeepbanerjee2003. I highly recommend clicking on the commit button that appears in my comment. It will speed things up and you might get a Github badge for it.

**Comment by subhadeepbanerjee2003** [comment]
@CBID2 now can you check if the changes reflected?

**Comment by CBID2** [comment]
Your turn @Anmol-Baranwal

**Comment by Anmol-Baranwal** [comment]
> Still needs improvement @subhadeepbanerjee2003. I highly recommend clicking on the commit button that appears in my comment. It will speed things up and you might get a Github badge for it.

I can directly add it when merging😅

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Comment by subhadeepbanerjee2003** [comment]
Thanks @Anmol-Baranwal

**Pull request merged**